e2e/helpers/juju-helpers.ts

```diff
@@ -0,0 +1,153 @@
+import { execSync } from "child_process";
+import fs from "node:fs/promises";
+
+import {
+  type Resource,
+  CloudAccessTypes,
+  ModelAccessTypes,
+  ResourceType,
+} from "../fixtures/setup";
+
+export class JujuHelpers {
+  private cleanupStack: Resource[];
+
+  constructor(cleanupStack?: Resource[]) {
+    this.cleanupStack = cleanupStack || [];
+  }
+
+  async removeModel(
+    modelName: string,
+    owner: string | undefined = process.env.USERNAME,
+  ) {
+    execSync(`juju destroy-model ${owner}/${modelName} --force --no-prompt`);
```

Comment:
```suggestion
    await exec(`juju destroy-model ${owner}/${modelName} --force --no-prompt`);
```

Same with all the other `execSync`...